secant/Util/ErrorWrapper.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ErrorWrapper.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 30.03.2022.
+//
+
+import Foundation
+
+/// Equatable wrapper for the errors, used in TCA's states
+struct ErrorWrapper: Equatable {
```

Comment:
Ideally errors are handled in the store and the state is updated based on the specific error with context so that the user of the store doesn't have to deal with `any error` but rather specifically known errors that are useful to the UI if needed.  I've noticed this is only used in one place that isn't marked with a TODO, should both places be marked with a todo for handling errors?  Is this wrapper temporary if we do implement error handling?  If so, we should probably have a TODO here on this type then as well I would think.

Reply:
There are going to be changes to Error handling so to avoid doing "future development" I wouldn't go around this too much

Reply:
I don't think you'll need the reflection or wrapper at all, you'll probably just need to define your own Error types that are equatable.  Throwing any error from underlying dependencies probably won't be useful to the UI or the consumer of the store.

Reply:
simplified for now with `.failed` state